.travis.yml

```diff
@@ -26,7 +26,7 @@ before_install:
   - ./bin/travis-before-install.sh
 
 install:
-  - npm install
+  - npm install --verbose
```

Comment:
This propagates down to node bindings so we get full output from meson and verbose ninja